### ISSUE TITLE
[D3D11] Titan Quest Anniversary Edition wrong refresh rate video mode (on DisplayPort)

### ISSUE THREAD

**Issue by eLPuSHeR** [doitsujin/dxvk#2607]
Titan Quest Anniversary Edition (with all DLCs). Running under W10 19044.1645 x64. Using latest DXVK release. It works great for my RX480 card (60Hz) but for some weird reason instead of 1920x1080x165Hz, it sets 50Hz mode.

**Reports with no log files will be ignored.**

### Software information
Titan Quest Anniversary Edition (with all DLCs).

### System information
- GPU: i7 5960-x (8 cores, 16 threads)
- Driver: nVidia 512.15 (GTX 1070)
- Wine version: N/A
- DXVK version: 1.10.1

### Apitrace file(s)
- Put a link here

### Log files
- d3d9.log: 
- d3d11.log: 
[TQ_d3d11.log](https://github.com/doitsujin/dxvk/files/8481809/TQ_d3d11.log)
- dxgi.log: 
[TQ_dxgi.log](https://github.com/doitsujin/dxvk/files/8481811/TQ_dxgi.log)

**Comment by doitsujin** [comment]
Yes, this is a somewhat common issue on Windows and I don't know why it happens. Any help in resolving it without breaking wine would be appreciated.

**Comment by lextra2** [comment]
On windows, just delete all resolutions from your EDID with [CRU](https://www.monitortests.com/forum/Thread-Custom-Resolution-Utility-CRU), so the only one left is 1920x1080@165hz (inside CTA-861)

**Comment by WinterSnowfall** [comment]
To add some context here (and [a trace](https://drive.proton.me/urls/TZBKA04BXW#acGPe9mFieq7)), it looks like the game confuses the numerator with the denominator and sets them the other way around. That is causing dxvk to pick the lowest supported refresh rate, but that's similar to native behavior.

WineD3D somehow still picks the correct one.

**Comment by WinterSnowfall** [comment]
On setups where this is a problem or can't be addressed otherwise, the newly introduced `d3d9/dxgi.forceRefreshRate` option can be used to get the game on the preferred mode.
